src/test/java/wbooks/userdata/userdata-schema.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "#number? _ == userId",
```

Comment:
abstract this logic of the schema. This validation must be added in the feature.

Reply:
Updated the schema and added a set id on the feature.
I think that the set line could be done better but couldn't find an alternative in karate docs 😭  

Btw, I'm not sure if you meant to not update the schema at all haha, I could add the userId validation in another step on the scenario but not sure if it could be done in a single match 🤔